pom.xml

```diff
@@ -212,6 +127,13 @@
     <maven.build.timestamp.format>yyyyMMdd-HHmm</maven.build.timestamp.format>
     <jacoco-maven-plugin.version>0.7.6.201602180812</jacoco-maven-plugin.version>
     <mockito.version>1.10.19</mockito.version>
+    
+    <final.war.name>cbioportal-${project.version}</final.war.name>
+
+	<db.test.driver>com.mysql.jdbc.Driver</db.test.driver>
+	<db.test.url>jdbc:mysql://127.0.0.1:3306/cgds_test</db.test.url>
+    <db.test.username>cbioportal</db.test.username>
+    <db.test.password>cbioportal</db.test.password>
```

Comment:
I understand this adds a parameter to set the test database, but I meant more that there are several ways we specify default values now:

1. specified in the `pom.xml` through `variablename:default-value` syntax. 
2. In `.properties.EXAMPLE`
3. In `GlobalProperties.java`
4. This PR seems yet another way to give a default value directly in the pom

I'm not sure what approach is best, but it would be nice to make some guidelines on this to follow.